src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/pages/Dashboard.vue

```diff
@@ -73,6 +73,9 @@
             <v-col v-if="showTransferCard" cols="12" sm="6">
               <TransferCard />
             </v-col>
+            <v-col v-if="showIcraCard" cols="12" sm="6">
+              <IcraCard />
+            </v-col>
```

Comment:
same comment as below We should use the <ConditionalRenderICRA> component. I think this will make it easier in the future to remove the flag code since it'll be the same everywhere in the application.